icu4c/source/.clang-format

```diff
@@ -0,0 +1,11 @@
+# © 2020 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+
+---
+Language: Cpp
+BasedOnStyle: LLVM
+IndentWidth: 4
+ColumnLimit: 105
+AllowShortBlocksOnASingleLine: false 
+AllowShortIfStatementsOnASingleLine: true 
```

Comment:
I propose adding this:
AllowShortFunctionsOnASingleLine: false

Reply:
Okay, take your time :)

Reply:
The code has lots of examples of one-line {} function bodies (I didn't try to compare to others), so I withdraw my request to add this here. :) LGTM.